Package.swift

```diff
@@ -0,0 +1,37 @@
+// swift-tools-version: 5.9
```

Comment:
This should be able to be 5.10 I think?

Reply:
I was using Xcode 15.2 which didn't have 5.10. I have now updated. Is there something we specifically need / want in 5.10?

Reply:
nope, nothing specific

Reply:
I've updated it